ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -166,6 +170,10 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
     -> BlockQuery (ShelleyBlock proto era)
              (Delegations (EraCrypto era), SL.RewardAccounts (EraCrypto era))
 
+  GetFilteredVoteDelegatees
+    :: Set (SL.Credential 'SL.Staking (EraCrypto era))
+    -> BlockQuery (ShelleyBlock proto era) (VoteDelegatees (EraCrypto era))
```

Comment:
My guess is that this query will only be used in Conway, can you confirm that? In that case, it can be experimental, and you can remove `ShelleyNodeToClientVersion9` again and instead let it be enabled by the existing `ShelleyNodeToClientVersion8` as mentioned by @jasagredo.

Also very minor: Can you add this constructor to the end :smile: